README.md

```diff
@@ -47,34 +47,55 @@ If you're using the Strimzi User Operator, the values for these environment vari
-* `KAFKA_VERSION`: version of the Kafka cluster. Default `2.8.0`.
-* `SARAMA_LOG_ENABLED`: enables the Sarama client logging. Default `false`.
-* `VERBOSITY_LOG_LEVEL`: verbosity of the tool logging. Default `0`. Allowed values 0 = INFO, 1 = DEBUG, 2 = TRACE.
-* `TLS_ENABLED`: if the canary has to use TLS to connect to the Kafka cluster. Default `false`.
-* `TLS_CA_CERT`: TLS CA certificate, in PEM format, to use to connect to the Kafka cluster. When this parameter is empty (default behaviour) and the TLS connection is enabled, the canary uses the system certificates trust store. When a TLS CA certificate is specified, it is added to the system certificates trust store.
-* `TLS_CLIENT_CERT`: TLS client certificate, in PEM format, to use for enabling TLS client authentication against the Kafka cluster. Default empty.
-* `TLS_CLIENT_KEY`: TLS client private key, in PEM format, to use for enabling TLS client authentication against the Kafka cluster. Default empty.
-* `TLS_INSECURE_SKIP_VERIFY`:  if the underneath Sarama client has to verify the server's certificate chain and host name. Default `false`.
-* `SASL_MECHANISM`: mechanism to use for SASL authentication against the Kafka cluster. Supported are `PLAIN`, `SCRAM-SHA-256` and `SCRAM-SHA-512`. Default empty.
-* `SASL_USER`: username for SASL authentication against the Kafka cluster when one of `PLAIN`, `SCRAM-SHA-256` or `SCRAM-SHA-512` is used. Default empty.
-* `SASL_PASSWORD`: password for SASL authentication against the Kafka cluster when one of `PLAIN`, `SCRAM-SHA-256` or `SCRAM-SHA-512` is used. Default empty.
-* `CONNECTION_CHECK_INTERVAL_MS`: it defines how often the tool has to check the connection with brokers (in ms). Default `120000`.
-* `CONNECTION_CHECK_LATENCY_BUCKETS`: buckets of the histogram related to the broker's connection latency metric (in ms). Default `100,200,400,800,1600`.
-* `STATUS_CHECK_INTERVAL_MS`: it defines how often (in ms) the tool updates internal status information (i.e. percentage of consumed messages) to expose outside on the corresponding HTTP endpoint. Default `30000`.
-* `STATUS_TIME_WINDOW_MS`: it defines the sliding time window size (in ms) in which status information are sampled. Default `300000`
+When running the Strimzi canary tool, it is possible to configure different aspects by using the environment variables listed in the following table.
+
+In addition, certain aspects can be overridden dynamically at runtime from a JSON configuration file.  Where this is possible, a field name is provided in the
+table. The configuration file described in more detail the next section.
```

Comment:
Too many spaces between sentences. Line-breaks in the middle of sentence seem weird - we normally have sentence per line or paragraph per line.